plumber/ppl/test/retention/record_deleter_queries_test.exs

```diff
@@ -0,0 +1,125 @@
+defmodule Ppl.Retention.RecordDeleterQueriesTest do
+  use ExUnit.Case
+
+  alias Ppl.EctoRepo
+  alias Ppl.PplRequests.Model.PplRequests
+  alias Ppl.Retention.RecordDeleterQueries
+
+  setup do
+    Test.Helpers.truncate_db()
+    :ok
+  end
+
+  describe "delete_expired_batch/1" do
```

Comment:
Can you add one integration test here that does the following:
- starts a pipeline execution and waits for it to finish successfully, [example](https://github.com/semaphoreio/semaphore/blob/main/plumber/ppl/test/grpc/server_test.exs#L511-L518) 
- deletes the pipeline using this new query
- asserts that records are deleted from all related pipeline tables similar to [this](https://github.com/semaphoreio/semaphore/blob/main/plumber/ppl/test/ppl_requests/model/ppl_requests_queries_test.exs#L136), and that related blocks are deleted in the block_repo database